lib/components/note-preview/index.tsx

```diff
@@ -118,12 +117,16 @@ export const NotePreview: FunctionComponent<Props> = ({
           const taskIndex = Array.prototype.indexOf.call(allTasks, node);
 
           let matchCount = 0;
-          const content = note.content.replace(/[\ue000|\ue001]/g, (match) =>
-            matchCount++ === taskIndex
-              ? match === '\ue000'
-                ? '\ue001'
-                : '\ue000'
-              : match
+
+          const content = note.content.replace(
+            /(- \[x\]|- \[ \])/g,
```

Comment:
either by adding an `i` flag here or using `\[xX\]` we should allow for either case of `x`

Reply:
Good change.

Reply:
https://github.com/Automattic/simplenote-electron/pull/2461